src/globals.c

```diff
@@ -54,6 +54,7 @@ char *mounted_dirs = NULL;
 char *bundle_to_add = NULL;
 struct timeval start_time;
 char *state_dir = NULL;
+int skip_diskspace_check = false;
```

Comment:
`int ... false`? we have `bool` types.

Reply:
bool types don't work with getopt long options :( The reference I used:  https://www.gnu.org/software/libc/manual/html_node/Getopt-Long-Options.html

That said, I should change this to  "= 0" at least because setting an int to false is incorrect.

Will change this.